ui/components/app/wallet-overview/coin-overview.tsx

```diff
@@ -0,0 +1,472 @@
+
+  const portfolioEvent = () => {
+    trackEvent({
+      category: MetaMetricsEventCategory.Navigation,
+      event: MetaMetricsEventName.MMIPortfolioButtonClicked,
+    });
+  };
+
+  const stakingEvent = () => {
+    trackEvent({
+      category: MetaMetricsEventCategory.Navigation,
+      event: MetaMetricsEventName.MMIPortfolioButtonClicked,
+    });
+  };
+
+  const renderInstitutionalButtons = () => {
+    return (
+      <>
+        <IconButton
+          className={`${classPrefix}-overview__button`}
+          Icon={<Icon name={IconName.Stake} color={IconColor.primaryInverse} />}
+          label={t('stake')}
+          onClick={() => {
+            stakingEvent();
+            global.platform.openTab({
+              url: `${mmiPortfolioUrl}/stake`,
+            });
+          }}
+        />
+        {mmiPortfolioEnabled && (
+          <IconButton
+            className={`${classPrefix}-overview__button`}
+            Icon={
+              <Icon name={IconName.Diagram} color={IconColor.primaryInverse} />
+            }
+            label={t('portfolio')}
+            onClick={() => {
+              portfolioEvent();
+              global.platform.openTab({
+                url: mmiPortfolioUrl,
+              });
+            }}
+          />
+        )}
+      </>
+    );
+  };
+  ///: END:ONLY_INCLUDE_IF
+
+  ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
+  const { openBuyCryptoInPdapp } = useRamps();
+  ///: END:ONLY_INCLUDE_IF
+
+  return (
+    <WalletOverview
+      balance={
+        <Tooltip
+          position="top"
+          title={t('balanceOutdated')}
+          disabled={!balanceIsCached}
+        >
+          <div className={`${classPrefix}-overview__balance`}>
+            <div className={`${classPrefix}-overview__primary-container`}>
+              {balance ? (
+                <UserPreferencedCurrencyDisplay
+                  style={{ display: 'contents' }}
+                  className={classnames(`${classPrefix}-overview__primary-balance`, {
+                    [`${classPrefix}-overview__cached-balance`]: balanceIsCached,
+                  })}
+                  data-testid={`${classPrefix}-overview__primary-currency`}
+                  value={balance}
+                  type={
+                    showPrimaryCurrency(
+                      isOriginalNativeSymbol,
+                      useNativeCurrencyAsPrimaryCurrency,
+                    )
+                      ? PRIMARY
+                      : SECONDARY
+                  }
+                  ethNumberOfDecimals={4}
+                  hideTitle
+                />
+              ) : (
+                <Spinner
+                  color="var(--color-secondary-default)"
+                  className="loading-overlay__spinner"
+                />
+              )}
+              {balanceIsCached ? (
+                <span className={`${classPrefix}-overview__cached-star`}>*</span>
+              ) : null}
```

Comment:
```suggestion
              {balanceIsCached && (
                <span className={`${classPrefix}-overview__cached-star`}>*</span>
              )}
```

Reply:
Couldn't merge your suggestion (code was outdated), but I did it manually, thanks!